src/User/Password/PasswordAlgorithm.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MsgPhp\User\Password;
+
+/**
+ * @author Roland Franssen <franssen.roland@gmail.com>
+ */
+final class PasswordAlgorithm
+{
+    private const DEFAULT_LEGACY_TYPE = 'sha512';
```

Comment:
[Symfony does N iterations over the hashing](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Security/Core/Encoder/MessageDigestPasswordEncoder.php#L57), it also does a base64 in the end.

Reply:
well.. SF is the standard infra anyway (which relates to your comment above yes). Not really aiming to make this full featured or so.. Is such a strategy required for us?

Reply:
Let's say someone is migrating from the default symfony, to an implementation provided by msgphp, it probably wouldn't be able to calculate the password due to lack of iterations and base64. Could probably fix that, question is if it's worth it.

Reply:
ok. we could create 2 more legacy options for that :+1:  But relates to above.. more options will be silently ignored for the SF implem.

Reply:
Fixed.